sig-testing/README.md

```diff
@@ -11,8 +11,10 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 Interested in how we can most effectively test Kubernetes. We're interested specifically in making it easier for the community to run tests and contribute test results, to ensure Kubernetes is stable across a variety of cluster configurations and cloud providers.
 
 ## Meetings
-* [Tuesdays at 13:00 PT (Pacific Time)](https://zoom.us/my/k8s.sig.testing) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=13:00&tz=PT%20%28Pacific%20Time%29).
-
+* [Main SIG-Meeting, Tuesdays at 13:00 PT (Pacific Time)](https://zoom.us/my/k8s.sig.testing) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=13:00&tz=PT%20%28Pacific%20Time%29). 
+Meeting notes and Agenda can be found [here](https://bit.ly/k8s-sig-testing-notes).
+Meeting recordings can be found [here](https://bit.ly/k8s-sig-testing-videos).
+* [Testing commons subproject, Wednesday at 11:00 PT (Pacific Time)](https://zoom.us/my/k8s.sig.testing) (bi-weekly) [Convert to your timezone](http://www.thetimezoneconverter.com/?t=13:00&tz=PT%20%28Pacific%20Time%29) 
```

Comment:
This timing works for me.

Reply:
This is quite difficult for me as I'm in the UK. Ideal for me would be 9am Pacific time, since that's still within my working day (it's 5pm UK time).  10am Pacific time (6pm UK time) means I'm staying late at work on Wednesdays, but I can do it. Later than that clashes with my family commitments.

Reply:
Same as @totherme I'm in the UK, so basically I have the same time constrains. I might be a bit more flexible and could maybe join some of the meetings at 11:00 PT.

Reply:
I have meetings from 8:30-10:30pst on wed.  I could do earlier than 8:30, but that might preclude attendance from other pst attendees.

Reply:
Before 8:30pst (16:30 UK) on Wednesdays is fine for me. I'd also be happy to pick another day.

Reply:
Does 8:00 PST work for folks?

Reply:
8:00 PST (16:00 UK) works for me.

Reply:
Let's tentatively plan for tomorrow at 7:30 PST and we can work out logistics from there.  

Zoom URL = https://zoom.us/j/2419653117 , and I'll talk with @castrojo about getting a calendar time setup once we get everyone else on the same page.